config/environments/production.rb

```diff
@@ -119,11 +119,6 @@
     config.logger = ActiveSupport::TaggedLogging.new(logger)
   end
 
-  # Enable JSON-style logging
-  config.logstasher.enabled = true
-  config.logstasher.logger = Logger.new(Rails.root.join("log/#{Rails.env}.json.log"))
-  config.logstasher.suppress_app_log = false
-
```

Comment:
I've given this a try on integration and all seems to be working fine 👍

Reply:
Thank you. :)